extract/extract/extractors/zalando.py

```diff
@@ -131,37 +128,24 @@ def __get_sustainability_info(
         beautiful_soup (BeautifulSoup): Parsed HTML
         title_attr (str): HTML attr of title
         description_attr (str): HTML attr of description
-        headline (str, optional): Headline on webpage. Defaults to
-            "Dieser Artikel erfüllt die folgenden Nachhaltigkeits-Kriterien:".
+        certificate_attr (str): HTML attr of certificate section
 
     Returns:
         Dict[str, str]: `dict` with name and description of found sustainability information
     """
 
-    # this area is hidden by default, so we need to find the right one
-    hidden_areas = [
-        div
-        for div in beautiful_soup.find_all("div", attrs={"style": "max-height:0"})
-        if headline in div.text
-    ]
-
-    if hidden_areas:
-        sustainability_info_parsed = hidden_areas[0]
-
-        titles = sustainability_info_parsed.find_all(attrs={"data-testid": title_attr})
-        descriptions = sustainability_info_parsed.find_all(attrs={"data-testid": description_attr})
+    sustainability_info_parsed = beautiful_soup.find("div", attrs={"data-testid": certificate_attr})
+    if not sustainability_info_parsed:
+        return {}
```

Comment:
In this case it would be much clearer if the condition is "positive":
```python
if sustainability_info_parsed:
  .. do things and return

else:
  .. do nothing and return {} in this case.
```